### DISCUSSION TITLE
Update outline via docker

### DISCUSSION THREAD

**Discussion by pranaykotas** [outline/outline#5189] [Self Hosting]
Hi everyone, I have done a v0.68 self-hosted install via docker on digitalocean. I'm 17 versions behind the latest one. How do I upgrade to the latest one? What commands should I use?

**Comment by pranaykotas** [comment]
Hi everyone, could someone help this newbie?

**Reply by tommoor** [reply]
Please see [the docs](https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t#h-updating)
